### DISCUSSION TITLE
Block client based on time of day

### DISCUSSION THREAD

**Discussion by nikeshjauhari** [AdguardTeam/AdGuardHome#4085] [General]
Hi,

Any idea how we can block a client IP based on the time of day?
example: Block IP (192.168.1.10 from 11:00 PM to 7:00 AM every day)

Thanks,
Nikesh

**Comment by ppfeufer** [comment]
Didn't know I needed this, but now I do :-)

**Reply by nikeshjauhari** [reply]
if AdGuardHome team implements this feature, 75% of pi-hole users will migrate to AdGuardHome.

**Comment by ainar-g** [comment]
Hello.  This is already a feature request #951.
